csrc/scheduler/mma_utils.cpp

```diff
@@ -22,17 +22,18 @@ namespace mma_utils {
 bool generateSharedMemoryEpilogueHeuristics(
     const MatMulTileOptions& gemm_tile,
     const int smem_double_buffer_stage,
-    const MmaDataTypes& data_types) {
+    const MmaDataTypes& data_types,
+    const bool ignore_occupancy_drop) {
   const auto properties = at::cuda::getCurrentDeviceProperties();
   const size_t device_smem_limit = properties->sharedMemPerBlockOptin;
+  // 1024 bytes are reserved for kernel launch overhead
+  const size_t shared_memory_overhead = 1024;
```

Comment:
Should we use this?
https://docs.nvidia.com/cuda/cuda-runtime-api/structcudaDeviceProp.html#structcudaDeviceProp_15a1f6049e9e9fd8af0d2a8a5df289ab7

Reply:
yes!